src/patchkernel/patch_kernel_parallel.cpp

```diff
@@ -1520,17 +1552,7 @@ void PatchKernel::partitioningCleanup()
 */
 bool PatchKernel::isPartitioned() const
```

Comment:
Should this wrapper function isPartitioned used at the place of isCommunicatorSet function? In this case is it better to use it everywhere even in PatchKernel and its derived classes to avoid the procreation of re-coding around, in case of change in the wrapper function?

Reply:
Done.

Please note that some functions that handle the communicator may be simplified when the deprecated partitioning functions (the ones that take the communicator as argument) are be removed.